docs/data/base/components/slider/slider.md

```diff
@@ -1,193 +1,243 @@
-  <span
-    aria-hidden="true"
-    data-index="2"
-    class="base-Slider-markLabel"
-    style="left: 100%;"
-    >100</span
-  >
-  <span class="base-Slider-thumb">
-    <input />
-  </span>
-</span>
-```
-
-:::info
-Both the `mark` and `markLabel` slots have corresponding `*Active` classes that are applied conditionally.
-:::
-
-### Custom structure
-
-Use the `slots` prop to override the root or any other interior slot:
+Once you have the package installed, import the component.
 
 ```jsx
-<Slider slots={{ root: 'div', thumb: 'div' }} />
+import * as Slider from '@base_ui/react/Slider';
 ```
 
-:::info
-The `slots` prop is available on all non-utility Base components.
-See [Overriding component structure](/base-ui/guides/overriding-component-structure/) for full details.
-:::
+### Anatomy
 
-Use the `slotProps` prop to pass custom props to internal slots.
-The following code snippet applies a CSS class called `my-rail` to the rail slot:
+Sliders are implemented using a collection of related components:
 
-```jsx
-<Slider slotProps={{ rail: { className: 'my-rail' } }} />
+- `<Slider.Root />` is a top-level component that wraps the other components.
+- `<Slider.Output />` renders the value of the slider.
+- `<Slider.Control />` renders the click/touch area that contains the track and thumb.
+- `<Slider.Track />` renders the visible rail on the `Control` along which the thumb(s) can be moved.
+- `<Slider.Indicator />` renders the filled portion of the track which represents the value(s).
+- `<Slider.Thumb />` renders the element that can be moved along the track to change the value.
+
+```tsx
+<Slider.Root>
+  <Slider.Output />
+  <Slider.Control>
+    <Slider.Track>
+      <Slider.Indicator />
+      <Slider.Thumb />
+    <Slider.Track/>
+  </Slider.Control>
+</Slider.Root>
 ```
 
-### Usage with TypeScript
+## Value
+
+### Default value
 
-In TypeScript, you can specify the custom component type used in the `slots.root` as a generic parameter of the unstyled component.
-This way, you can safely provide the custom root's props directly on the component:
+When Slider is uncontrolled, the `defaultValue` prop sets the initial value of the component.
 
 ```tsx
-<Slider<typeof CustomComponent> slots={{ root: CustomComponent }} customProp />
+function App() {
+  return (
+    <Slider.Root defaultValue={50}>
+      <Slider.Output />
+      <Slider.Control>
+        <Slider.Track>
+          <Slider.Indicator />
+          <Slider.Thumb />
+        <Slider.Track/>
+      </Slider.Control>
+    </Slider.Root>
+  );
+}
 ```
 
-The same applies for props specific to custom primitive elements:
+### Controlled
+
+When Slider is uncontrolled, the `value` prop holds the numerical value(s), and two callbacks are provided for when the value changes:
+
+- `onValueChange` is called when the value is changing while the thumb is being moved along the control area
+- `onValueCommitted` is called when thumb stops moving and `pointerup` is triggered
```

Comment:
So this prop doesn't apply to keyboard arrow keys?

Reply:
It does! It also fires on `keydown`, I've updated this sentence